.gitlab-ci.yml

```diff
@@ -8,12 +8,14 @@ include:
     - if: $CI_COMMIT_BRANCH == 'master'
   variables: &tmpl_master_variables
     SLS_STAGE: production
+    LOG_LEVEL: error
```

Comment:
Can we move this variable assignments to `serverless.js` like we did to the `DEBUG` variable? https://github.com/yldio/asap-hub/blob/master/serverless.js#L51

Reply:
there:
https://github.com/yldio/asap-hub/compare/72608c553d667092afcf3caccefa3f3e2fb6143d..09d56336b2b5bb59a987ece7a32dce5d9aa2440d

Reply:
Perf! 👍🏽